docs/src/views/CustomComponents.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from 'react'
+import * as cx from 'classnames'
+import * as _ from 'lodash'
+import { NavLink } from 'react-router-dom'
+import { Header } from 'semantic-ui-react'
+import {
+  Button,
+  Divider,
+  Provider,
+  createComponent,
+  ComponentSlotStyle,
+  ComponentVariablesInput,
+} from '@stardust-ui/react'
+
+import DocPage from '../components/DocPage/DocPage'
+import ExampleSnippet from '../components/ExampleSnippet/ExampleSnippet'
+
+interface StyledButtonProps {
+  className?: string
+  styles?: ComponentSlotStyle
+  variables?: ComponentVariablesInput
+  children?: React.ReactNodeArray | React.ReactNode
+}
+
+const StyledButton: React.SFC<StyledButtonProps> = createComponent<StyledButtonProps, {}>({
+  displayName: 'StyledButton',
+  render({ stardust, ...props }) {
+    const { classes } = stardust
+    const componentProps = _.pick(props, 'children')
+    return <button className={cx(props.className, classes.root)} {...componentProps} />
+  },
+})
```

Comment:
I think that we should omit the usage of `lodash` in such basic examples:

```js
const StyledButton: React.SFC<StyledButtonProps> = createComponent<StyledButtonProps, {}>({
  displayName: 'StyledButton',
  render({ stardust, ...props }) {
    const { classes } = stardust

    return <button {...rest} className={cx(props.className, classes.root)} />
  },
})
```

As I see, my code does the same thing or no?

Reply:
Something like this:
```jsx
const StyledButton: React.SFC<StyledButtonProps> = createComponent<StyledButtonProps, {}>({
  displayName: 'StyledButton',
  render({ stardust, className, children }) {
    const { classes } = stardust
    const componentProps = { children }

    return <button className={cx(className, classes.root)} {...componentProps} />
  },
})
```

Reply:
```diff
-return <button className={cx(className, classes.root)} {...componentProps} />
+return <button className={cx(className, classes.root)}>{children}</button>
```

👍